app/scripts/modules/core/src/projects/ProjectDisplayRenderer.tsx

```diff
@@ -0,0 +1,35 @@
+import { IProject } from 'core/domain';
+import {
+  AbstractBaseResultRenderer,
+  ITableColumnConfigEntry
+} from 'core/search/searchResult/AbstractBaseResultRenderer';
+
+import './project.less';
+
+export class ProjectDisplayRenderer extends AbstractBaseResultRenderer<IProject> {
```

Comment:
Since these renderers are effectively static, can't we just eschew the whole `extends AbstractBase` thing and just have this configuration be a const passed into a `ResultRenderer<T>.new(config)` or something (didn't spend too much time thinking of architecture)? Seems weird having a static instance with a public accessor (The Singleton pattern feels weird in JavaScript)

Reply:
@christopherthielen and I have talked about a better way to architect this and the persisted property stuff in the -nflx branch.  He doesn't think this should keep us from committing this currently and I want to get this in front of our stakeholders.  I can work on it as soon as it's released as they'll have some time to play.

Reply:
That makes sense.